<launch>

	<node pkg="ur5e_control" type="ur5e_controller" name="ur5e_controller_node" />
	<node pkg="surgery_sim" type="plan_listener" name="plan_listener" >
		<param name="plan_type" value="planner"/>
	</node>
	<node pkg="surgery_sim" type="pedal_controller" name="pedal_controller" />
	<node pkg="surgery_sim" type="point_counter" name="point_counter" />
	<node pkg="surgery_sim" type="task_space_traj_reset" name="task_space_traj_node" />
	<node pkg="surgery_sim" type="frame_tf_broadcaster" name="frame_broadcast_node" />
	<node pkg="surgery_sim" type="listener" name="listener_node" >
		<param name="move_scale" value="0.5"/>
	</node>
	<node pkg="surgery_sim" type="switch_node_no_ff" name="switch_node_no_ff" />
	<node pkg="surgery_sim" type="overlay_broadcast" name="overlay_broadcast" />
	<node pkg="surgery_sim" type="user_overlay" name="user_overlay" >
		<param name="source" value="plan"/>
		<param name="mode" value="crop"/>
	</node>

</launch>

<!-- 

source options - "click"; "plan"; "path"
mode options - "resize"; "crop"; "full"
plan_type options - "coded"; "clicked"; "planner"; "model"
move_scale options - any float not greater an 3.0

-->
